Tests/SentryTests/Integrations/Performance/CoreData/TestCoreDataStack.swift

```diff
@@ -99,3 +100,40 @@ class TestCoreDataStack {
         }
     }
 }
+
+class TestNSManagedObjectContext: NSManagedObjectContext {
+
+    var inserted: Set<NSManagedObject>?
+    var updated: Set<NSManagedObject>?
+    var deleted: Set<NSManagedObject>?
+
+    override var insertedObjects: Set<NSManagedObject> {
+        inserted ?? []
+    }
+
+    override var updatedObjects: Set<NSManagedObject> {
+        updated ?? []
+    }
+
+    override var deletedObjects: Set<NSManagedObject> {
+        deleted ?? []
+    }
+
+    init() {
+        super.init(concurrencyType: .mainQueueConcurrencyType)
+    }
+
+    override init(concurrencyType ct: NSManagedObjectContextConcurrencyType) {
+        super.init(concurrencyType: ct)
+    }
+
+    required init?(coder: NSCoder) {
+        super.init(coder: coder)
+    }
+
+    override var hasChanges: Bool {
+        return ((inserted?.count ?? 0) > 0) ||
+        ((deleted?.count ?? 0) > 0) ||
+        ((updated?.count ?? 0) > 0) || super.hasChanges
```

Comment:
Had to add `|| super.hasChanges` for SentryCoreDataTrackerTests.test_Save() to work again, because it doesn't rely on the subclass properties.